service-monitor/src/main/java/com/yahoo/vespa/service/health/ApacheHttpClient.java

```diff
@@ -57,7 +57,7 @@ public long getKeepAliveDuration(HttpResponse response, HttpContext context) {
                     }
                 };
 
-        return HttpClients.custom()
+        return VespaHttpClientBuilder.createWithBasicConnectionManager()
                 .setKeepAliveStrategy(keepAliveStrategy)
                 .setConnectionManager(connectionManager)
```

Comment:
This would override the connection manager & ssl connection socket factory!?

Reply:
Good catch! Fixed.